GAE/src/com/gallatinsystems/survey/domain/Question.java

```diff
@@ -62,6 +62,7 @@
     private Long questionGroupId;
     private Long surveyId;
     private String questionId;
```

Comment:
Perhaps we can mark this one as deprecated and put a note of the transformation to `variableName`.